Configs/.config/hypr/windowrules.conf

```diff
@@ -70,4 +70,6 @@ windowrulev2 = float,class:^(blueman-manager)$
 windowrulev2 = float,class:^(nm-applet)$
 windowrulev2 = float,class:^(nm-connection-editor)$
 windowrulev2 = float,class:^(org.kde.polkit-kde-authentication-agent-1)$
-
+windowrulev2 = opacity 0.80 0.80,class:^(org.freedesktop.impl.portal.desktop.gtk)$
+windowrulev2 = opacity 0.80 0.80,class:^(org.freedesktop.impl.portal.desktop.kde)$
```

Comment:
why kde portal?? it would be killed by hyprland session due to [this](https://github.com/prasanthrangan/hyprdots/blob/main/Configs/.config/hyprdots/scripts/resetxdgportal.sh).

Reply:
Yeah I wondered too 😄. I'll remove this part